integration/emr/alluxio-emr.sh

```diff
@@ -261,8 +263,7 @@ main() {
   fi
 
   # Create user
-  sudo groupadd alluxio -g 600
-  sudo useradd alluxio -u 600 -g 600
+  id -u alluxio &>/dev/null || sudo useradd alluxio
```

Comment:
You're gonna need to spell this one out for me

Reply:
runs both commands. succeeds as long as one of them passes. So either we create the user or the user already exists.